### PR TITLE
[KOGITO-4135] Improve SEO of DMN.new and BPMN.new

### PR DESCRIPTION
**Pull request by manaswinidas**
JIRA link: https://issues.redhat.com/browse/KOGITO-4135

### CODE REVIEW

**Review by ljmotta — APPROVED** [review]
LGTM! Thanks for this PR @manaswinidas

File: packages/online-editor/static/index.html [hunk 1]
```diff
@@ -21,20 +21,27 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta
+      name="description"
+      content="DMN.new and BPMN.new let you draw Decision Model and Notation(DMN), Predictive Model Markup Language(PMML) and Business Process Model and Notation(BPMN) diagrams for your business processes"
```
Comment (tomasdavidorg) [inline packages/online-editor/static/index.html]: Just small suggestion. I would add space before the left parenthesis and add the dot at the end of the sentence.
```suggestion
      content="DMN.new and BPMN.new let you draw Decision Model and Notation (DMN), Predictive Model Markup Language (PMML) and Business Process Model and Notation (BPMN) diagrams for your business processes."
```

**Review by tomasdavidorg — APPROVED** [review]
@manaswinidas Thanks 🙂

**Review by tiagobento — APPROVED** [review]

**Comment by tiagobento** [comment]
@tomasdavidorg  @domhanak Hi guys! We have consistent failures of the Chrome and VS Code extension tests on this PR for some reason. Might be a good one to try and see what can be causing these failures. Can you please take a look?

**Comment by tomasdavidorg** [comment]
@tiagobento Chrome test fails are caused by the incompatibility of Chrome driver and Chrome browser (This branch is not rebased with the master branch where the driver is updated.) Desktop test fail on windows is caused by some instability of the github ci machine. Desktop fail on the Mac is probably something related with connection to selenium server... I will take a look more on this one.

**Comment by domhanak** [comment]
VSCode on mac-os fails because the dependencies are not resolved

**Comment by tiagobento** [comment]
@tomasdavidorg @domhanak Ok! Thx! Merging this PR as test failures are unrelated.

**Pull request merged**